src/types/User.ts

```diff
@@ -35,5 +35,13 @@ export interface User {
   kidsMode: boolean;
   anonMode: boolean;
   enteredVenueIds?: string[];
+  status?: string;
   myPersonalizedSchedule?: MyPersonalizedSchedule;
 }
+
+export enum UserStatus {
+  online = "online",
+  busy = "busy",
+  offline = "offline",
+  custom = "custom",
```

Comment:
As per the comment above, remove offline and online

Reply:
https://github.com/sparkletown/sparkle/pull/1384/commits/8c87d4d1376bd2174c883f9f92d1bfcf9a2ac85e